<launch>
  <rosparam command="load" file="/opt/home_services/src/home_services/cfg/hs_params.yaml" />
  <node name="finder_node" pkg="torrent_search" type="finder_node.py" args="--debug"/>

<!-- Comment -->
</launch>
